pkg/types/repo_conf.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import "os"
+
+func InsecureSkipTLSForRepo() (insecureSkipTls bool) {
+	insecureSkipTls = false
+	if os.Getenv("TRIVY_INSECURE") != "" {
+		insecureSkipTls = true
+	}
+	return insecureSkipTls
+}
```

Comment:
I think we can move this parameter to a cli option.
https://github.com/aquasecurity/trivy/blob/0518d2785fb6bb0ebec4ad6cef053d94cbad03ab/pkg/types/docker_conf.go#L17

We can define `--insecure` and `TRIVY_INSECURE` both.
https://github.com/aquasecurity/trivy/blob/884daff4296f33b9653caaf7051544c9ec01c43e/pkg/commands/app.go

Reply:
Done